tests/unit/simple_model.py

```diff
@@ -161,7 +161,7 @@ def step(self, closure=None):
                 state = self.state[p]
                 if len(state) == 0:
                     state['integer_step'] = 0
-                    state['tensor_step'] = torch.zeros(1)
```

Comment:
@jeffra, FYI.

Reply:
Really excellent find, this had been bugging me for a while :)